elide-integration-tests/src/test/groovy/com/yahoo/elide/tests/FilterIT.groovy

```diff
@@ -867,6 +889,41 @@ class FilterIT extends AbstractIntegrationTestInitializer {
         }
     }
 
+    @Test
+    public void testGetBadRelationshipNameWithNestedFieldFilter() {
```

Comment:
We should have tests for:
1. an invalid root with the new filter method
2. an invalid leaf